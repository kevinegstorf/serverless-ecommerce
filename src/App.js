import React, { Component } from 'react';
import { BrowserRouter, Router, Route, Switch, Link} from "react-router-dom";
import HomePage from './pages/Homepage/HomePage';
import ProductsPage from './pages/ProductsPage/ProductsPage';

export default class  App extends Component {
  render() {
    return (
      <Switch>
        <Route exact path="/" component={HomePage}/>
        <Route path="/products" component={ProductsPage}/>
      </Switch>
    );
  }
}
