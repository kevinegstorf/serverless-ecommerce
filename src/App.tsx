import React, { Component } from 'react';
import { BrowserRouter, Route, Switch} from "react-router-dom";

import HomePage from './pages/Hompage/Homepage';
import ProductsPage from './pages/ProductPage/ProductPage';

export default class  App extends Component {
  render() {
    return (
      <BrowserRouter>
        <Switch>
          <Route exact path="/" component={HomePage}/>
          <Route path="/products" component={ProductsPage}/>
        </Switch>
      </BrowserRouter>
      );
  }
}