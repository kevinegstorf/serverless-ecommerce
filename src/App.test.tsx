import React from 'react';
import ReactDOM from 'react-dom';
import App from './App';
import { render } from 'react-testing-library';

it('renders without crashing', () => {
  const div = document.createElement('div');
  ReactDOM.render(<App />, div);
  ReactDOM.unmountComponentAtNode(div);
});

test('snapshot test', () => {
  const container = render(<App/>)
  expect(container).toMatchSnapshot();
});
