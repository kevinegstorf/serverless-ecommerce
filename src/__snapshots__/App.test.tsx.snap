// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot test 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div>
        <div
          class="css-obaqkn"
        >
          <ul
            class="css-10e31a0"
          >
            <li>
              HOME
            </li>
            <li>
              CONTACT
            </li>
            <li>
              DELIVERY
            </li>
            <li>
              TERMS
            </li>
            <li>
              <input />
            </li>
            <li>
              LOGIN
            </li>
          </ul>
        </div>
        <div>
          <h1>
            shirt shop
          </h1>
        </div>
      </div>
    </div>
  </body>,
  "container": <div>
    <div>
      <div
        class="css-obaqkn"
      >
        <ul
          class="css-10e31a0"
        >
          <li>
            HOME
          </li>
          <li>
            CONTACT
          </li>
          <li>
            DELIVERY
          </li>
          <li>
            TERMS
          </li>
          <li>
            <input />
          </li>
          <li>
            LOGIN
          </li>
        </ul>
      </div>
      <div>
        <h1>
          shirt shop
        </h1>
      </div>
    </div>
  </div>,
  "debug": [Function],
  "getAllByAltText": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllBySelectText": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getAllByValue": [Function],
  "getByAltText": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getBySelectText": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "getByValue": [Function],
  "queryAllByAltText": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllBySelectText": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryAllByValue": [Function],
  "queryByAltText": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryBySelectText": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "queryByValue": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
